integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/testsupport/util/ElasticsearchTckBackendFeatures.java

```diff
@@ -248,6 +248,28 @@ public boolean supportsExplicitRefresh() {
 
 	@Override
 	public boolean supportsVectorSearch() {
-		return false;
+		return isActualVersion(
+				esVersion -> !esVersion.isLessThan( "8.0" ),
+				osVersion -> !osVersion.isLessThan( "2.0" ),
+				osVersion -> true
```

Comment:
```suggestion
				es -> !es.isLessThan( "8.0" ),
				os -> !os.isLessThan( "2.0" ),
				aoss -> true
```

Also, you're sure Amazon OpenSearch Serverless supports vector search?

Reply:
I cannot see it here https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-genref.html
but then there's https://aws.amazon.com/opensearch-service/serverless-vector-engine/